meson.build

```diff
@@ -0,0 +1,76 @@
+project(
+  'libvfio-user',
+  'c',
+  version: '0.1',
```

Comment:
should this maybe be 0.0.1?

Reply:
Sure, whatever you want the versioning number to say.